src/SWP/Bundle/CoreBundle/Theme/Configuration/TenantableConfigurationSourceFactory.php

```diff
@@ -68,6 +68,9 @@ public function initializeSource(ContainerBuilder $container, array $config)
             ]),
         ]);
 
+        $container->setParameter('sylius.theme.configuration.filename', $config['filename']);
```

Comment:
`swp.theme.configuration.filename` and `sylius.theme.configuration.default_directory` looks much better as this is our custom configuration that we use in `CoreBundle` so it shouldnt have `sylius` prefix.